### PR TITLE
[TT-1718] Update Havoc Docs

### PR DESCRIPTION
**Pull request by kalverra**
Also removes old jekyll docs.

---
**Below is a summarization created by an LLM (gpt-4-0125-preview). Be mindful of hallucinations and verify accuracy.**

## Why
The changes streamline and update documentation, improve code readability, and enhance functionality within the chaos testing framework. Enhancements to chaos experiment monitoring and observability through Grafana annotations and structured logging also stand out, contributing to a more efficient chaos testing process in Kubernetes environments.

## What
- **.gitignore**
  - Removed specific `docs` entries in favor of a generalized `book/docs/` path. This change consolidates documentation-related ignores into a more streamlined pattern.
- **book/README.md**
  - Updated the documentation to reflect the usage of `mdBook` for Chainlink Testing Framework Docs, including development setup instructions and how to run the mdBook locally.
- **book/src/libs/havoc.md**
  - Significantly condensed the documentation for the `havoc` package, focusing on its purpose, features, requirements, and setup for chaos testing with Kubernetes and Chaos Mesh, including installation instructions and a brief on active monitoring and Grafana annotations.
- **Deleted files under `docs` directory**
  - Entire `docs` directory including various markdown files, images, and custom stylesheets have been removed, indicating a shift or cleanup of the project's documentation strategy.
- **havoc/README.md**
  - Minor modifications to align with the broader documentation updates, focusing on the `havoc` package.
- **havoc/chaos_helper.go**
  - Clarified the comment for `WaitForAllChaosRunning` to better describe its functionality.
- **havoc/chaos_listener.go**
  - Added a comment explaining the `ChaosListener` interface, improving code documentation and readability.
- **havoc/go.mod**
  - Updated dependencies which could include both enhancements and critical fixes.
- **havoc/range_grafana_annotator.go**
  - Added a placeholder function `hey()`, the purpose of which is unclear from the provided diff.
- **havoc/single_line_grafana_annotator.go**
  - Added a detailed comment for `SingleLineGrafanaAnnotator`, improving documentation within the code.
- **havoc/single_line_grafana_annotator_test.go**
  - Introduced a new test file to exemplify the creation and setup of a chaos experiment, highlighting the use of `NewChaos` with a pod chaos example, and demonstrating the integration of logging and annotation functionalities.

### CODE REVIEW

File: havoc/single_line_grafana_annotator_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		"http://grafana-instance.com",
+		"grafana-access-token",
+		"dashboard-uid",
+		Logger,
+	)
+	chaos.AddListener(logger)
+	chaos.AddListener(annotator)
+
+	chaos.Create(context.Background())
+	// Output:
```
Comment (skudasov) [inline havoc/single_line_grafana_annotator_test.go:38]: Is it urging me to write more code to handle output? Not sure what to do

File: havoc/range_grafana_annotator.go [hunk 2]
```diff
@@ -239,3 +239,6 @@ func (l RangeGrafanaAnnotator) OnScheduleDeleted(chaos Schedule) {
 	}
 	l.chaosMap[chaos.Object.GetName()] = res.ID
 }
+
+func hey() {
```
Comment (skudasov) [inline havoc/range_grafana_annotator.go:243]: ✋

**Review by skudasov — APPROVED** [review]

File: book/src/libs/havoc.md [hunk 3]
```diff
@@ -1,211 +1,41 @@
...
-`ChaosLogger` leverages the zerolog library to provide structured, queryable logging of chaos events. It automatically logs key lifecycle events such as creation, start, pause, and termination of chaos experiments, including detailed contextual information.
+## Installation
 
-Instantiate `ChaosLogger` and register it as a listener to your chaos experiments:
-
-```
-logger := havoc.NewChaosLogger()
-chaos.AddListener(logger)
+```sh
+go get -u github.com/smartcontractkit/chainlink-testing-framework/havoc
```
Comment (skudasov) [inline book/src/libs/havoc.md:22]: Maybe just `go get`? Updating packages is on user, maybe I don't need it

File: havoc/single_line_grafana_annotator_test.go (hunk 1 above)
Comment (kalverra) [reply havoc/single_line_grafana_annotator_test.go:38]: No, this is just how [Go examples verify output when run](https://go.dev/blog/examples) (my least favorite part of how it works). This is just saying that we don't expect any output from this code. I dunno if the chaos experiment would have some dummy logging we could print out?

File: havoc/range_grafana_annotator.go (hunk 2 above)
Comment (kalverra) [reply havoc/range_grafana_annotator.go:243]: 👋

File: book/src/libs/havoc.md (hunk 3 above)
Comment (kalverra) [reply book/src/libs/havoc.md:22]: I'd argue it's unnecessary most of the time. if you're familiar with Go, this isn't exactly new.

**Pull request merged**